test/built-ins/Temporal/Duration/compare/relativeto-sub-minute-offset.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2021 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.duration.compare
+description: relativeTo string accepts an inexact UTC offset rounded to hours and minutes
+features: [Temporal]
+---*/
+
+const duration1 = new Temporal.Duration(0, 0, 0, 31);
+const duration2 = new Temporal.Duration(0, 1);
+
+let relativeTo = "2000-01-01T00:00+00:45[+00:44:30.123456789]";
+assert.sameValue(Temporal.Duration.compare(duration1, duration2, { relativeTo }), 0, "rounded HH:MM is accepted in string");
+
+relativeTo = "2000-01-01T00:00+00:44:30[+00:44:30.123456789]";
+assert.throws(RangeError, () => Temporal.Duration.compare(duration1, duration2, { relativeTo }), "no other rounding is accepted for offset");
```

Comment:
I cannot see how this RangeError should be thrown.
the "+00:44:30.123456789" part in relativeTo is TimeZoneUTCOffsetName in the grammar, which is totally ignored by the spec text except syntax check .